Packs/EWS/ReleaseNotes/2_0_5.md

```diff
@@ -2,4 +2,4 @@
 #### Integrations
 ##### EWS O365
 - Added the **Mark fetched emails as read** parameter to the integration instance configuration.
```

Comment:
Why this file is here? revert the changes

Reply:
from master (not my change), I added extra space by mistake